### ISSUE TITLE
Not an issue, really, but some questions.

### ISSUE THREAD

**Issue by Lord-Memester** [stealthmonkey99/OctoPrint-PWMBuzzer#12]
What buzzer would you suggest? Additionally, would a speaker work?  

Could a script or other program be run to output the audio via the jack on the pi?

Just things I'd like to know. Good luck fixing the other issues I saw here!

**Comment by stealthmonkey99** [comment]
Hi @Lord-Memester - I've tried a few buzzers and each one sounds uniquely awful 😆 so it's hard to pick one that I'd "recommend".  Here are three that all worked with just wiring up the negative to ground and positive to a 3.3V GPIO pin:

- https://www.adafruit.com/product/1898
- https://www.adafruit.com/product/1891
- https://www.amazon.com/gp/product/B07MRJKHCQ (when I was done practicing soldering, I removed the buzzer to use with this plugin project)

Many speakers will require more than the 3.3V that I've been using off a GPIO pin, but if you have a spare one handy it's worth a shot (let me know how it goes).  You might also experiment with amplifying/using 5V like I saw something about in here: https://raspberrypi.stackexchange.com/questions/61547/different-frequencies-with-piezo-buzzer-python.

As to the last one... great question, but unfortunately the plugin won't work through the audio jack.
